libbeat/autodiscover/appenders/config/config.go

```diff
@@ -20,6 +20,8 @@ package config
 import (
 	"fmt"
 
+	"github.com/pkg/errors"
```

Comment:
Did you mean "errors" or "github.com/pkg/errors". Just asking as I'm seeing this wrong import a lot recently.

Reply:
https://github.com/pkg/errors is the one that gives me `errors.Wrap`, I think this is correct?